clang/lib/Sema/SemaChecking.cpp

```diff
@@ -286,6 +286,9 @@ static bool BuiltinFunctionStart(Sema &S, CallExpr *TheCall) {
   if (S.checkArgCount(TheCall, 1))
     return true;
 
+  if (TheCall->getArg(0)->containsErrors())
```

Comment:
`containsErrors` only works when we have error-recovery (`RecoveryExpr`s) eanbled.  Can you write a test/make sure the behavior is acceptable as well for `-fno-recovery-ast`